init/init.go

```diff
@@ -39,7 +40,7 @@ func writeConfig(fs afero.Fs, config *v2.Config) error {
 //Init reads user console input and generates a fogg.yml file
 func Init(fs afero.Fs) error {
 	project, region, bucket, table, profile, owner := userPrompt()
-	config := config.InitConfig(project, region, bucket, table, profile, owner, AWSProviderVersion)
+	config := config.InitConfig(project, region, bucket, table, profile, owner, AWSProviderVersion, foggVersion)
```

Comment:
I feel like if you're just passing in a constant value to this function it might just be worth it to move that constant over to the config package and reference it directly from there without it having to be a function argument.